eng/common/scripts/apiview-helper.ps1

```diff
@@ -0,0 +1,56 @@
+
+
+function Map-LanguageToAPiview-Filter($language)
+{
+    $lang = $language
+    # Update language name to match those in API cosmos DB. Cosmos SQL is case sensitive and handling this within the query makes it slow.
+    if($lang -eq 'javascript'){
+        $lang = "JavaScript"
+    }
+    elseif ($lang -eq "dotnet"){
+        $lang = "C#"
+    }
+    elseif ($lang -eq "java"){
+        $lang = "Java"
+    }
+    elseif ($lang -eq "python"){
+        $lang = "Python"
+    }
+    return $lang
+}
+
+function Check-ApiReviewStatus($packageName, $language)
+{
+  # Get API view URL and API Key to check status
+  Login-AzAccount
```

Comment:
This adds a new set of dependencies for folks which not requires that they have the Az powershell cmdlets which a lot of folks like don't. Given that we require az cli already could these be implemented using az cli instead?

Reply:
done.